lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -73,6 +73,7 @@
                         @navManager={{navManager}}
                         @draftManager={{draftManager}}
                         @onSubmitRedirect={{this.onSubmitRedirect}}
+                        @isMobileView={{this.showMobileView}}
```

Comment:
so in mobile view, is the right-nav never even shown (because everything is in the horizontal navbar instead), or is there a button to expand it?

if it's never shown, it might be cleaner to wrap this `<RightNav>` invocation in `{{#if (not this.showMobileView)}}`, so it doesn't render at all unless needed

if the right-nav can expand on mobile, we might have to make the `tabindex=-1` toggle accordingly, which feels potentially messy

(as always, would be nice if the right-nav and top-nav were the same component with media-query'd styles... alas)

Reply:
+1 at wrapping the whole rightnav in a conditional. When it's in mobile-view, the rightnav is moved off the screen in favor of buttons on the horizontal navbar.